spyder/widgets/switcher.py

```diff
@@ -805,12 +805,13 @@ def create_symbol_example_switcher(sw):
     sw.add_item(title=_('another symbol'))
 
 
-def create_example_switcher(main=None):
+def create_example_switcher(main=None, item_styles=None,
```

Comment:
@dalthviz this file should not be modified on this PR, right?

Reply:
I think I changed this to test the Switcher without a parent and with different styles. Should I revert this?

Reply:
Yes, I mean that this is the PR for the completion improvements, not the switcher one, so maybe you did the changes in the wrong branch 🤷‍♂ ?

Reply:
Oh yep totally right!